docs/source/en/index.mdx

```diff
@@ -12,18 +12,13 @@ specific language governing permissions and limitations under the License.
 
 # 🤗 Transformers
 
-State-of-the-art Machine Learning for PyTorch, TensorFlow and JAX.
+State-of-the-art Transformers models in PyTorch, TensorFlow, and JAX for research and production.
 
-🤗 Transformers provides APIs to easily download and train state-of-the-art pretrained models. Using pretrained models can reduce your compute costs, carbon footprint, and save you time from training a model from scratch. The models can be used across different modalities such as:
+🤗 Transformers is dedicated to providing inference, finetuning, and pretraining for Transformer-based models on text, computer vision, speech, and multimodal tasks. Each model is defined by three classes - a configuration, a model, and a preprocessor - which can be initialized from pretrained instances stored on the [Hugging Face Hub](https://huggingface.co/models). The library also offers convenient helper tools like a simple pipeline to use a model for inference on a given task and a feature-complete training loop optimized for 🤗 Transformers models.
```

Comment:
This is not an improvement compared to before IMO. In particular, I liked the list.

Reply:
I also prefer the former, to be honest

Reply:
What do you think about bringing back most of the list and combining it with a simplified version of the changes introduced here? I think the list is useful for showcasing what the library is capable of, but I think it is also important to briefly describe the library's goals and what it is focused on. I can take out the part about the `pipeline` and `Trainer` here.

Reply:
Again, the changes focus too much on Transformer-based models. The goal of the library is perfectly stated in the sentence you are changing (previous version).